wcfsetup/install/files/lib/system/database/table/column/ObjectIdDatabaseTableColumn.class.php

```diff
@@ -16,7 +16,7 @@
  */
 final class ObjectIdDatabaseTableColumn
 {
-    public static function create($name): IDatabaseTableColumn
+    public static function create(string $name): IntDatabaseTableColumn
```

Comment:
The return type was intentionally selected to be unspecific, to possibly allow changing the type later (e.g. to BIGINT). Whether that actually is feasible is a different story.